### ISSUE TITLE
Better statistics for URL columns

### ISSUE THREAD

**Issue by mariosasko** [huggingface/datasets-server#2539]
URLs are stored as strings, so their stat info is their (string) lengths. We could make this info more informative by replacing the lengths with URL domain distribution (or some other stat/metric?).

**Comment by polinaeterna** [comment]
agree, string lengths is pretty much useless here. domain distribution sounds reasonable, i don't what else we can count...

**Comment by polinaeterna** [comment]
@mariosasko can you give some examples of datasets where this might be useful?

**Comment by mariosasko** [comment]
E.g., these ones: https://huggingface.co/search/full-text?q=fetch_images&type=dataset (datasets with image URLs that are used to download the actual images)
